test/tasks/mock/MockMultisigTask.sol

```diff
@@ -48,7 +48,7 @@ contract MockMultisigTask is MultisigTask {
         }
```

Comment:
The build is failing here, we need this import: `import {VmSafe} from "forge-std/Vm.sol";`